src/main/java/org/sagebionetworks/web/client/jsinterop/KeyFactory.java

```diff
@@ -3,14 +3,15 @@
 import java.util.List;
 import jsinterop.annotations.JsConstructor;
 import jsinterop.annotations.JsType;
+import org.sagebionetworks.web.client.jsinterop.reactquery.QueryKey;
 
 @JsType(isNative = true, namespace = "SRC.SynapseQueries")
 public class KeyFactory {
 
   @JsConstructor
   public KeyFactory(String accessToken) {}
 
-  public native List<?> getDownloadListBaseQueryKey();
+  public native QueryKey getDownloadListBaseQueryKey();
```

Comment:
Small improvement to our JsInterop types/classes - Our `KeyFactory` now returns a `QueryKey`